### PR TITLE
Switch to protobuf3

### PR DESCRIPTION
**Pull request by Tristramg**
I also fixed some minor clippy warnings. If you prefer that I do a separate pull request, just tell me!

### CODE REVIEW

File: src/groups.rs [hunk 1]
```diff
@@ -173,33 +173,37 @@ pub struct Relations<'a> {
...
+                            member: match t {
+                                NODE => NodeId(m).into(),
+                                WAY => WayId(m).into(),
+                                RELATION => RelationId(m).into(),
+                            },
+                            role: make_string(role as usize, self.block),
+                        })
+                    } else {
+                        None
                     }
```
Comment (TeXitoi) [inline src/groups.rs:200]:
```suggestion
                    t.enum_value().map(|t| Ref {
                        member: match t {
                            NODE => NodeId(m).into(),
                            WAY => WayId(m).into(),
                            RELATION => RelationId(m).into(),
                        },
                        role: make_string(role as usize, self.block),
                    })
```

**Review by TeXitoi — CHANGES_REQUESTED** [review]
Great work, thanks!

Only a minor stylish remark.

**Comment by Tristramg** [comment]
Thank you for the quick review.
I rebased on the right remote and applied your suggestion

**Comment by TeXitoi** [comment]
v0.17.0 published, thanks!

**Comment by TeXitoi** [comment]
10 years of osmpbfreader!

**Pull request merged**